setup_lib.sh

```diff
@@ -27,6 +27,6 @@ copy_config() {
 
 update_config() {
   sed -i "s/RaspiWiFi Setup/${CONFIG_SSID:-RaspiWiFi Setup}/" /etc/raspiwifi/raspiwifi.conf
-  [[ $AUTO_CONFIG = "y" ]] && sed -i "s/auto_config=0/auto_config=1/" /etc/raspiwifi/raspiwifi.conf
+  [ "x$AUTO_CONFIG" = "xy" ] && sed -i "s/auto_config=0/auto_config=1/" /etc/raspiwifi/raspiwifi.conf
```

Comment:
Just curious, why is the `x` prepended on the answer here? Why resolve to `xy` instead of just `y`?

Reply:
I made this to take care of edge cases with some versions of bash, when the tested variable is empty, more details here: https://stackoverflow.com/questions/174119/why-do-shell-script-comparisons-often-use-xvar-xyes

Reply:
Ah, that makes perfect sense. Thanks.